### ISSUE TITLE
Post on solve

### ISSUE THREAD

**Issue by ghost** [drunohazarb/4chan-captcha-solver#20]
4chan-x's post on captcha solve and auto-load captcha are broken with current captcha and solvers (understandably)
Can you please add a checkmark that immediately posts when captcha is solved? since only the solver knows when it's solved, and 4chan-x has no way to know that.

**Comment by JonseyJones** [comment]
+1 for this feature.

**Comment by cnozxjun1** [comment]
<img width="777" alt="Screenshot 2024-01-30 at 14 27 05" src="https://github.com/drunohazarb/4chan-captcha-solver/assets/158166724/39ad1ccd-9483-40bd-9738-8f273776ea3b">
they clapped my account in like 10 minutes 

anyway here's the code 

https://files.catbox.moe/0e1289.js

```
// ==UserScript==
// @name         Auto Click Submit
// @namespace    sneed
// @match        https://boards.4channel.org/*
// @match        https://boards.4chan.org/*
// @match        https://sys.4chan.org/*
// @match        https://sys.4channel.org/*
// @version      2024-01-30
// @description  try to take over the world!
// @author       Me
// @grant        none
// ==/UserScript==

(function() {
    'use strict';
    window.addEventListener("4chan-captcha-solver:solved", () => {
        const submit = document.querySelector("#file-n-submit > input:nth-child(3)");
        submit.click();
    })
})();
```

**Comment by cnozxjun1** [comment]
I'm not schizophrenic right? Can anyone else see those messages?

**Comment by JonseyJones** [comment]
> I'm not schizophrenic right? Can anyone else see those messages?

Script doesn't work.

**Comment by cnozxjun1** [comment]
You need the updated script from here https://files.catbox.moe/0e1289.js and that snippet in another user script and also 4chanx

**Comment by JonseyJones** [comment]
> You need the updated script from here https://files.catbox.moe/0e1289.js and that snippet in another user script and also 4chanx

I already had the @version     1.4.9 wasm from https://github.com/drunohazarb/4chan-captcha-solver/blob/master/4chan-captcha-solver-wasm.user.js, what is the difference between that and yours ?
Only works with yours.

**Comment by cnozxjun1** [comment]
I added an event to it. It's just my concept, up to Bruno if he agrees with my vision

**Comment by JonseyJones** [comment]
> I added an event to it. It's just my concept, up to Bruno if he agrees with my vision

>https://files.catbox.moe/0e1289.js 

Maybe it's just me but it stopped working.

**Comment by JonseyJones** [comment]
@drunohazarb  can you make a proper addition to the script like the "save" option?
@cnozxjun1 's  is not proper, it works only sometimes, for me anyway.

**Comment by ghost** [comment]
> @drunohazarb can you make a proper addition to the script like the "save" option? @cnozxjun1 's is not proper, it works only sometimes, for me anyway.

yeah although I appreciate @cnozxjun1 's effort, I'd rather have a proper addition to the script instead of something like this (that'll inevitably break as soon as anything changes, if it works at all)

**Comment by trip54654** [comment]
This is a different thing, but does anyone have a script that puts the input focus back to the captcha text field after 4chan's java script focuses the slider?

**Comment by ghost** [comment]
@drunohazarb any comment?

**Comment by gir489returns** [comment]
@cnozxjun1 Can you modify the script to require a new CAPTCHA and post when the 60 second post timer finishes?

**Comment by JonseyJones** [comment]
Hey @drunohazarb, bro, here's how I see it, either solve it or close it. If the feature is not needed/wanted bin the thread.
Great job with the latest ver of the script btw as far as I'm concerned you can close the other threads/issues too since I don't get those problems, switching IPs and browsers, fully flushing the history etc., even on mobile browsers, for me it's >95% good.
Cheers and much respect!

**Comment by realyukii** [comment]
@drunohazarb by the way, what is the different between wasm and non wasm version? what should I choose?

**Comment by gir489returns** [comment]
> @drunohazarb by the way, what is the different between wasm and non wasm version? what should I choose?

WebAssembly turns the JS code into raw hardware code so it can run on the hardware, whereas the non WASM version runs within the browser exclusively. You want the WASM version because it's much faster. Unless you like waiting 5 seconds for it to solve the CAPTCHA.
